CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Dependencies
 - Bump `netty` from 4.1.91.Final to 4.1.93.Final ([#7901](https://github.com/opensearch-project/OpenSearch/pull/7901))
+- Bump `spock-core` from 1.3-groovy-2.5 to 2.1-groovy-2.5 ([#8119](https://github.com/opensearch-project/OpenSearch/pull/8119))
```

Comment:
I think the commit version and changelog version need to be bumped. 

```suggestion
- Bump `spock-core` from 1.3-groovy-2.5 to 2.3-groovy-2.5 ([#8119](https://github.com/opensearch-project/OpenSearch/pull/8119))
```

Reply:
Ah, good spot. Fixed now.